index.rst

```diff
@@ -123,7 +130,7 @@ Here we include a quick analysis to verify that it really was 42 seconds and who
    Maximum duration of tracking: 929.5563025474548
```

Comment:
Not important, but I would round all those values with 2 or 3 decimals.

Reply:
done